### PR TITLE
I added a new comment.

### PR DESCRIPTION
**Pull request by chadningle**
Chad testing out the process for contributing to the chef recipes via github.

### CODE REVIEW

File: recipes/login_banner.rb [hunk 1]
```diff
@@ -34,6 +34,8 @@
 # can easily get this information by running the “uname –a” command once
 # they have logged in.
 
+# This is a new comment. -- Chad
```
Comment (isuftin) [inline recipes/login_banner.rb]: Yes, it is a new comment, Chad :D

Comment (cschroed-usgs) [inline recipes/login_banner.rb]: Isn't it the best?

Comment (chadningle) [inline recipes/login_banner.rb]: This is another comment.  -- Chad

Comment (isuftin) [inline recipes/login_banner.rb]: :+1:

Comment (isuftin) [inline recipes/login_banner.rb]: @chadningle  You should edit the motd file to reflect how you want it to look and submit that

**Comment by cschroed-usgs** [comment]
:shipit: -- We'll add motd modifications in a subsequent PR. This is just for demo purposes for now.

**Pull request merged**